File: solutions/go/grains/1/grains.go
package grains
import "fmt"

 func Square(number int) (uint64, error) {
     if (number < 1 || number > 64){
         return 0, fmt.Errorf("Value out of range ")
     }
    return 1 <<(number - 1) , nil
}

func Total() uint64 {
	var total uint64
    for i := 1 ; i <= 64 ; i++ {
        n,_ := Square(i)
        total += n
    }
 return total 
}

File: solutions/python/grains/1/grains.py
def square(number):
    if number < 1 or number > 64:
         raise ValueError("square must be between 1 and 64")
    return  1 <<(number - 1)


def total():
    value = 0
    for i in range(1, 65):
        value += square(i)
    return value
